### DISCUSSION TITLE
How frequently is OpenStreetMap data reprocessed?

### DISCUSSION THREAD

**Discussion by julcnx** [Project-OSRM/osrm-backend#6843] [Q&A] [answered]
Four days ago, I appended necessary tags to a narrow road (https://www.openstreetmap.org/way/550732347) in order to accurately reflect routing on narrow roads. However, as of now, there have been no observable changes to the routing (https://www.openstreetmap.org/directions?engine=fossgis_osrm_car&route=13.76208%2C100.49531%3B13.76218%2C100.49368#map=19/13.76203/100.49450).

It's worth noting that other similarly tagged ways are being avoided by the router (https://www.openstreetmap.org/directions?engine=fossgis_osrm_car&route=13.71773%2C100.52527%3B13.71850%2C100.52358#map=18/13.71891/100.52461).

I have a suspicion that the new tags I implemented have not yet undergone preprocessing. However, I recall reading somewhere that data is typically updated every two days. How can I determine the specific timeframe for the data being utilized by OSMR in a particular area?

**Comment by jcoupey** [comment, marked as answer]
From _Overview of routing data updates_ at the bottom of https://routing.openstreetmap.de/about.html: "The routing data are updated roughly every two days." That page includes a link to the data timestamps.

For debugging purposes or to test the impact of small local changes, you're probably better off running your own local instance on a small region.

**Comment by CAM-Gerlach** [comment]
It seems the routing data hasn't been updated for a month and a half (since May 7-9, depending on region), which is a bit more than two days. Has something gotten stuck with the update process?
